spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-require ‘coveralls’
-Coveralls.wear_merged!(‘rails’)
+require 'coveralls'
+Coveralls.wear_merged!('rails')
```

Comment:
This was fixed in another PR? Do you have the latest code?

Reply:
Fixed